moono/css/navbar-icons.less

```diff
@@ -91,3 +91,36 @@
 .elfinder .ui-resizable-handle.ui-resizable-e {
   cursor: e-resize;
 }
+
+/* select-all (fa-check-square) */
+.elfinder-button-icon-selectall:before {
+  content: '\f14a';
+}
+/* unselect-all (fa-check-square-o) */
+.elfinder-button-icon-selectnone:before {
+  content: '\f046';
+}
+/* invert selection (fa-arrows-h) */
+.elfinder-button-icon-selectinvert:before {
+  content: '\f07e';
+}
+/* preferences (fa-cog) */
+.elfinder-button-icon-preference:before {
+  content: '\f013';
+}
+/* fullscreen (fa-fullscreen) */
+.elfinder-button-icon-fullscreen:before {
+  content: '\f0b2';
+}
+/* undo (fa-ban) */
+.elfinder-button-icon-undo:before {
```

Comment:
Was this meant to be [this](https://fontawesome.com/icons/undo?style=solid) icon?

![screenshot 2018-10-07 at 20 44 53](https://user-images.githubusercontent.com/4363863/46586148-1c9e0280-ca72-11e8-9876-788c4b8049ff.png)

Reply:
Yes [fa-undo](https://fontawesome.com/v4.7.0/icon/undo) is better! Ok, i hope i find time in these days for build it